From bef3c812038119d321c93097133c5468c01034ec Mon Sep 17 00:00:00 2001
From: Spotlight <spotlight@joscomputing.space>
Date: Fri, 14 Oct 2022 21:14:29 -0500
Subject: [PATCH] Mark mac99 as having a built-in IDE

---
 src/qemu/qemu_domain.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/qemu/qemu_domain.c b/src/qemu/qemu_domain.c
index ee024d1..f9fe422 100644
--- a/src/qemu/qemu_domain.c
+++ b/src/qemu/qemu_domain.c
@@ -8740,7 +8740,8 @@ qemuDomainMachineHasBuiltinIDE(const char *machine,
     return qemuDomainMachineIsI440FX(machine, arch) ||
         STREQ(machine, "malta") ||
         STREQ(machine, "sun4u") ||
-        STREQ(machine, "g3beige");
+        STREQ(machine, "g3beige") ||
+        STREQ(machine, "mac99");
 }
 
 
-- 
2.37.3

